src/fu-tool.c

```diff
@@ -976,15 +982,25 @@ fu_util_update_all (FuUtilPrivate *priv, GError **error)
 		if (!fu_util_is_interesting_device (dev))
 			continue;
 		/* only show stuff that has metadata available */
-		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_SUPPORTED))
+		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE))
+			continue;
+		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_SUPPORTED)) {
+			/* TRANSLATORS: message letting the user know no device upgrade available due to missing on LVFS
+			* %1 is the device name */
+			g_printerr (_("• %s does not have firmware updates posted on any enabled remotes\n"), fwupd_device_get_name (dev));
 			continue;
+		}
 		if (!fu_util_filter_device (priv, dev))
 			continue;
 
 		device_id = fu_device_get_id (dev);
 		rels = fu_engine_get_upgrades (priv->engine, device_id, &error_local);
 		if (rels == NULL) {
-			g_printerr ("%s\n", error_local->message);
+			/* TRANSLATORS: message letting the user know no device upgrade available
+			* %1 is the device name */
+			g_printerr (_("• %s is updated to the latest version available\n"), fwupd_device_get_name (dev));
```

Comment:
I'm not sure about the message, but can't think of a better one.

Reply:
My issue is that it sounds like we did the update, when it might have been updated in Windows, or in the factory.

Reply:
How about

```
• %s is running the latest version available firmware

```

Reply:
`%s has the latest available firmware version`

Reply:
Adjusted